portals/publisher/src/main/webapp/source/src/app/components/Apis/Details/Comments/CommentOptions.jsx

```diff
@@ -149,6 +149,7 @@ class CommentOptions extends React.Component {
         const user = AuthManager.getUser();
         const username = Utils.getUserNameWithoutDomain(user.name);
         const canDelete = (comment.createdBy === username) || user.isAdmin();
+        const canReply = !(user.isCreator() && !user.isAdmin());
```

Comment:
We can write the same as `!user.isCreator() || user.isAdmin()`
It's more readable IMO. So this implies the creator and admin can't reply and the publisher can.

Reply:
I'll do the change